go/runtime/registry/config.go

```diff
@@ -128,7 +128,8 @@ func newRuntimeConfig(dataDir string) (map[common.Namespace]map[version.Version]
 			if runtimes[id] == nil {
 				runtimes[id] = make(map[version.Version]*runtimeHost.Config)
 			}
-			if _, ok := runtimes[id][bnd.Manifest.Version]; ok {
+			version := bnd.Manifest.GetComponentByID(component.ID_RONL).Version
```

Comment:
Maybe add a comment that the component cannot be nil because regular bundles are the ones that have the RONL component which was checked above.

Reply:
This was latter changed (moved to registry) so no need to add a comment.